docs/core_docs/docs/integrations/vectorstores/couchbase.mdx

```diff
@@ -0,0 +1,367 @@
+---
+hide_table_of_contents: true
+sidebar_class_name: node-only
```

Comment:
Is this causing the `node-only` on the sidebar. Can we try removing it to see if this helps?